### ISSUE TITLE
Ubuntu core

### ISSUE THREAD

**Issue by mudler** [kairos-io/kairos#387]
This is a card to track ubuntu-core being added in our flavors

**Comment by 3pings** [comment]
I do not see this enhancement in the roadmap.  Is there a timeframe on when the Ubuntu Core flavor will be available?

**Comment by mudler** [comment]
we need to first run a spike on it - from a first investigation there are neither official container images nor generic ISOs. 

I could find only generic ISOs for virtualized environments, but ISO generically speaking are not enough and would require quite few work to get them into a container form first. I'm more concerned if there is no apt to bring in any packages that we might need to make it bootable.

**Comment by bencorrado** [comment]
I don't think this is a practical objective.  Ubuntu Core is designed to be extremely locked down and only runs snaps, so none of the the agents or modifications will work. This would be a huge lift to implement and would likely end up with a Kairos snap running inside Ubuntu core and needing a custom snap store ($$$ to Canonical) or running in dangerous mode.

**Comment by tctlrd** [comment]
Lets close this topic.
